test/policies/proxy/proxy-through-proxy.test.js

```diff
@@ -0,0 +1,62 @@
+const gwHelper = require('../../common/gateway.helper');
+const cliHelper = require('../../common/cli.helper');
+const httpProxy = require('http-proxy');
+const http = require('http');
+const assert = require('assert');
+const request = require('superagent');
+
+describe('@e2e @proxy through proxy', () => {
+  const gatewayConfig = {
+    apiEndpoints: {
+      api: {
+        path: '/test'
+      }
+    },
+    policies: ['proxy'],
+    pipelines: {
+      pipeline1: {
+        apiEndpoints: ['api'],
+        policies: [{
+          proxy: {
+            action: { serviceEndpoint: 'backend' }
+          }
+        }]
+      }
+    }
+  };
+  const proxiedUrls = {};
+  let gw;
+  before('init', (done) => {
+    cliHelper.bootstrapFolder().then(dirInfo => {
```

Comment:
Would it be possible to write this test by avoiding to create a new gateway from scratch, with all modules to install? This would make increase the tests time significantly.

I'd rather try to spin up the gateway inline, in the same you've been doing with the [key auth test](https://github.com/ExpressGateway/express-gateway/blob/master/test/policies/keyauth/keyauth.test.js)

Reply:
It is an  integration test. I should place it into e2e folder.